app/components/floating_text_input_label/index.tsx

```diff
@@ -133,6 +160,7 @@ const FloatingTextInput = forwardRef<FloatingTextInputRef, FloatingTextInputProp
     const inputRef = useRef<TextInput>(null);
     const [animation] = useState(new Value(focusedLabel ? 1 : 0));
     const debouncedOnFocusTextInput = debounce(setIsFocusLabel, 500, {leading: true, trailing: false});
+    const theme = useTheme();
```

Comment:
no no no no no.. this component is used in the login flow screens that do not really use the theme from the context.